### PR TITLE
CommVault Guide

### PR DESCRIPTION
**Pull request by kashifzeeshan**
Guide

## What Changed?

Please list, at a high level, what changed in your branch. If you changed content, include the product, section, and version as applicable. **Please remove the user instructions including the examples before merging the PR.**

**Examples**

- Fixed typo in EPAS 13 epas_inst_linux guide
- Added more detail to ARK 3.5 getting started guide introduction
- Fixed broken link on `/supported-open-source/pgbackrest/05-retention_policy/` page

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: advocacy_docs/partner_docs/CommVaultGuide/03-SolutionSummary.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: 'Brief explanation of the solution and its purpose'
+---
+Commvault enables your business to streamline management of its continuously evolving data environment, whether the data is on-premises or in the cloud.
+
+Commvault PostgreSQL iDataAgent provides the flexibility to backup PostgreSQL, EDB Postgres Advanced Server and EDB Postgres Extended Server databases in different modes and restore them when needed. You can perform a full/log backup or restore of database servers/individual databases/archive logs at any time and have full control over the process.
+
+Managing your data means knowing that it is protected and being able to effectively report on success or failure.  Through an easy-to-use interface, you can quickly check on the progress of your jobs to ensure things are moving as expected.  You can also use pre-built reports in an on-demand fashion, or scheduled, to keep you in the know.
+
+<p align="center">
+  <img src="Images/CommvaultEPASArchitecture.png">
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/03-SolutionSummary.mdx:12]: Need to update image.  
using our DB server image and add Extended server.
https://drive.google.com/drive/folders/1IjrxALM2N0fssOulY2r2EnA0TzO9KLE1
See if the png file I created looks good.  If not you can use the source material in folder to create a new one.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Commvault Backup & Recovery'
+---
+
+Implementing Commvault Backup & Recovery with EDB Postgres Advanced Server requires the following components:
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:6]: Implementing Commvault Backup & Recovery with an EDB database requires the following components:

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+- EDB Postgres Advanced Server
+- Commvault Backup & Recovery software
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:9]: EDB Postgres Advanced Server or EDB Postgres Extended Server

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server instance.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:13]: A running EDB Postgres Advanced Server or EDB Postgres Extended Server instance.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+- Commvault Backup & Recovery installed.
+- EDB Postgres Advanced Server application path and library directory path (e.g. c:\Program Files\edb\as13\bin and c:\Program Files\edb\as13\lib).
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:15]: EDB Postgres Advanced Server or EDB Postgres Extended Server application path ....

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+- The login credentials used to access the EDB Postgres Advanced Server database.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:16]: .... access the EDB Postgres Advanced Server or EDB Postgres Extended Server database.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+- EDB Postgres Advanced Server archive log directory configured.
+
+## Configure Commvault Backup & Recovery for EDB Postgres Advanced Server
+
+### Core Setup Wizard
+1. Run the Core Setup Wizard from Commvault Backup & Recovery's Command Centre from the Machine where Commvault Backup & Recovery is installed. The wizard helps to set up a disk storage pool and to modify the server backup plan according to your requirements.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:22]: Command Centre - spelling
Command Center

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+2. Setup Storage Pool/Disk Storage: From the Welcome page, click Let's get started. On the Disk tab, in the Name box, enter a name for the storage pool.
+3. In the MediaAgent box, accept the default value.
+4. Click Local on the radio button for Type.
+5. In the Backup location box, click Browse to assign a path where backups will be stored.
+6. To enable deduplication on the storage, move the Use deduplication toggle key to  the right, and in the Deduplication DB location box, browse to select the path to  the deduplication database.
+7. To move to the next setup option, click Save
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:28]: add a period at end "."

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+4. In the Database server name box, enter the server name.
+5. In the Username and Password boxes, enter the credentials to connect to the server.
+6. From the Plan list, select the server plan created in step 8 under Core Server Wizard section for use with EDB Postgres Advanced Server.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd2.png">
+</p>
+
+7. Once the server is added, a Job will run to install client on the Server. The screen shots below display the process from Job creation to software installation.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:60]: a Job will run to install client on the Server.
change - added "a"
a Job will run to install a client on the Server.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+8. Creating a Server Backup Plan in Core Setup: In order to create a server backup plan in Core Setup, define where the data is stored and how often must the data undergo a backup.
+9. In the Plan name box,type the name of the plan e.g EPAS Server Plan.
+10. In the Backup destinations section, select the Storage and Retention Period.
+11. In the RPO section, select the Backup frequency and Start time for the backup to start as per the backup frequency.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/BackupPlanConf.png">
+</p>
+
+### Setup EDB Postgres Advanced Server
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:43]: Setup EDB Postgres Advanced Server in the Commvault Backup & Recovery's Command Center

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server instance.
+- Commvault Backup & Recovery installed.
+- EDB Postgres Advanced Server application path and library directory path (e.g. c:\Program Files\edb\as13\bin and c:\Program Files\edb\as13\lib).
+- The login credentials used to access the EDB Postgres Advanced Server database.
+- EDB Postgres Advanced Server archive log directory configured.
+
+## Configure Commvault Backup & Recovery for EDB Postgres Advanced Server
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:19]: Heading now
Configure Commvault Backup & Recovery for EDB Postgres Advanced Server or EDB Postgres Extended Server

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+### Core Setup Wizard
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:21]: Heading now
Setup a Disk Storage Pool and Database Server Backup Plan

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+9. In the Plan name box,type the name of the plan e.g EPAS Server Plan.
+10. In the Backup destinations section, select the Storage and Retention Period.
+11. In the RPO section, select the Backup frequency and Start time for the backup to start as per the backup frequency.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/BackupPlanConf.png">
+</p>
+
+### Setup EDB Postgres Advanced Server
+1. From the navigation pane, go to Protect > Databases.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:44]: Step 1
 1. From the Navigation pane of the Commvault Backup & Recovery's Command Center, go to Protect > Databases.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd5.png">
+</p>
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd6.png">
+</p>
+
+### Setup EDB Postgres Advanced Server Instance
+1. From the navigation pane, go to Protect > Databases > DB Instances.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:79]: Title now
Configure the  EDB Database Instances to backup and protect.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+11. In the RPO section, select the Backup frequency and Start time for the backup to start as per the backup frequency.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/BackupPlanConf.png">
+</p>
+
+### Setup EDB Postgres Advanced Server
+1. From the navigation pane, go to Protect > Databases.
+2. Click Add server.
+3. Select the database type for EDB Postgres Advanced server, which in this case is PostgreSQL.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:46]: Select the database type for EDB database, which in this case is PostgreSQL.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd1.png">
+</p>
+
+4. In the Database server name box, enter the server name.
+5. In the Username and Password boxes, enter the credentials to connect to the server.
+6. From the Plan list, select the server plan created in step 8 under Core Server Wizard section for use with EDB Postgres Advanced Server.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:54]: in step 8 under Setup a Disk Storage Pool and Database Server Backup Plan section for use with your EDB database.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+1. From the navigation pane, go to Protect > Databases > DB Instances.
+2. Click Add instance, and then select PostgreSQL.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/CreateInstance1.png">
+</p>
+
+3. From the Server name list, select a server where you want to create the new instance.
+4. In the Instance Name box, type the EDB Postgres Advanced Server instance name.
+5. From the Plan list, select the server plan created in step 8 under Core Server Wizard section for use with EDB Postgres Advanced Server.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:88]: in step 8 under Setup a Disk Storage Pool and Database Server Backup Plan section for use with your EDB database.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+6. Under Connection details, enter the following information.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:89]: Replace 
"EDB Postgres Advanced Server" with
with
"EDB database"
for the next three entries.

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 19]
```diff
@@ -0,0 +1,185 @@
+---
+title: 'Using Commvault Backup & Recovery'
+description: 'Walkthroughs of multiple Commvault Backup & Recovery usage scenarios'
+---
+
+How to backups and restore from EDB Postgres Advanced Server using Commvault Backup & Recovery.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:6]: How to backup and restore an EDB Database using Commvault Backup & Recovery.

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 20]
```diff
@@ -0,0 +1,185 @@
...
+
+## Using Commvault Backup & Recovery
+
+Commvault provides two methods of taking the backup from EDB Postgres Advanced Server and restoring it.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:10]: Commvault provides two methods of taking the backup from an EDB database and restoring it.

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 21]
```diff
@@ -0,0 +1,185 @@
...
+
+<p align="center">
+  <img  alt="Restore Job" src="Images/Dumprestore4.png">
+</p>
+
+<p align="center">
+  <img  alt="Restore Job" src="Images/Dumprestore5.png">
+</p>
+
+9. Once the Restore is completed successfully, login to the EDB Postgres Advanced Server Instance and check that restore operation recovered the data.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:99]: login to the EDB database and check that the restore operation recovered the data.  In our example below we connected to an EDB Postgres Advanced Server.

File: advocacy_docs/partner_docs/CommVaultGuide/06-CertificationEnvironment.mdx [hunk 22]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'Certification Environment'
+description: 'Overview of the certification environment used in the certification of Commvault Backup & Recovery'
+---
+
+|  &nbsp;    | &nbsp; |
+| ----------- | ----------- |
+| **Certification Test Date**      | January 24, 2022       |
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/06-CertificationEnvironment.mdx:8]: Change date.
June 16, 2022

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 23]
```diff
@@ -0,0 +1,185 @@
...
+Foreign-key constraints:
+    "department_employee_fk" FOREIGN KEY (deptno) REFERENCES tp_department_db(deptno)
+
+
+epas13_test=#
+epas13_test=#
+
+```
+
+### FSBasedBackupSet Backup and Restore
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:180]: delete heading

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 24]
```diff
@@ -0,0 +1,185 @@
...
+
+At this time there is a known issue with FSBased Restore that does not allow for proper restoration of the database. See the known issues section of the guide for more information.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:182]: Delete this info.  Will be a note at the top of this section of the guide.

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 25]
```diff
@@ -0,0 +1,185 @@
...
+description: 'Walkthroughs of multiple Commvault Backup & Recovery usage scenarios'
+---
+
+How to backups and restore from EDB Postgres Advanced Server using Commvault Backup & Recovery.
+
+## Using Commvault Backup & Recovery
+
+Commvault provides two methods of taking the backup from EDB Postgres Advanced Server and restoring it.
+1. DumpBasedBackupSet Backup and Restore
+2. FSBasedBackupSet Backup and Restore
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:12]: add a note
"At this time there is a known issue with FSBased Restore that does not allow for proper restoration of the database. See the known issues section of the guide for more information."

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 26]
```diff
@@ -0,0 +1,185 @@
...
+epas13_test=#
+
+```
+
+### FSBasedBackupSet Backup and Restore
+
+At this time there is a known issue with FSBased Restore that does not allow for proper restoration of the database. See the known issues section of the guide for more information.
+
+#### Known Issues:
+1. FSBaseBadkupSet Restore has an issue if the *:\Program files\edb has been deleted, after the restore is done, the permissions on the restored directories are not recovered, and it instead inherits the permissions from the parent directory which does not allow EDB Postgres Advanced Server services to start on the restored directory. We are working with Commvault to resolve the issue.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:185]: replace with
FSBaseBadkupSet Restore has an issue if the default edb directory for example *:\Program files\edb has been lost or deleted.  If this occurs then after a restore is performed, the permissions on the restored directories are not recovered. Instead the directory inherits the permissions from the parent directory which does not allow EDB Postgres Advanced Server services to start on the restored directory. We are working with Commvault to resolve the issue.

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**